assets/javascript/game.js

```diff
@@ -1,219 +1,95 @@
 $(document).ready(function(){
 
-			var wins = 0;
-			var losses = 0;
-			
-			// This is the sum of the guesses added together. 
-			var counter = 0;
+  var audioElement1 = document.createElement('audio');
+  audioElement1.setAttribute('src', 'https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.WAV');
 
-			var crystal1 = 0;
-			var crystal2 = 0;
-			var crystal3 = 0;
-			var crystal4 = 0;
+  var audioElement2 = document.createElement('audio');
+  audioElement2.setAttribute('src', 'https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.mp3');
 
-			var audioElement1 = document.createElement("audio");
-      		audioElement1.setAttribute("src", "https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.WAV");
+  var song = document.createElement('audio');
```

Comment:
I converted all your tab indentation to spaces for consistency. Totally fine to use tabs, just make sure you pick one and stick with it.